src/vs/workbench/contrib/terminal/common/scripts/shellIntegration-bash.sh

```diff
@@ -18,6 +18,15 @@ bash_major_version=${BASH_VERSINFO[0]}
 __vscode_disable_env_reporting="$VSCODE_DISABLE_ENV_REPORTING"
 unset VSCODE_DISABLE_ENV_REPORTING
 
+__vscode_shell_env_setting="$VSCODE_SHELL_ENV_SETTING"
+unset VSCODE_SHELL_ENV_SETTING
```

Comment:
These are the same thing aren't they? We only need one variable to come through to control this. On the TS side we can have several things feed into the decision but in the end only one variable should control env reporting.

Reply:
Yeap good point, its lot more clear this way!! --> https://github.com/microsoft/vscode/pull/240424/commits/ba0f8abf42dc9251586838751e6c63bd463e2097